src/main/resources/org/jenkinsci/plugins/workflow/libs/LibraryConfiguration/config.jelly

```diff
@@ -37,7 +37,7 @@ THE SOFTWARE.
     <f:entry field="allowVersionOverride" title="${%Allow default version to be overridden}">
         <f:checkbox default="true"/>
     </f:entry>
-    <f:entry field="includeInChangesets" title="${%Include @Library changes in job recent changes}">
+    <f:entry field="includeInChangesets" title="${%Include @Library changes in job recent changes, and changes to this library should trigger jobs that use it}">
```

Comment:
Would be good to check how this looks in the UI.

Reply:
Ok, yeah I think this part of the change should be reverted, since it makes things align way too far to the right on small screens. Perhaps #80 would change the appearance in such a way that it wouldn't cause everything to do that?

<img width="1096" alt="Screen Shot 2019-12-11 at 09 49 41" src="https://user-images.githubusercontent.com/1068968/70631816-c1c6e480-1bfb-11ea-9ec1-55179bf82186.png">

Reply:
Good point, I've reverted that change now @dwnusbaum